Volume calculations

juliacosma opened the pull request:
Prints to file called "calc-volumes.txt"

juliacosma committed:
Initial volume calculation

Prints to file called "calc-volumes.txt"

juliacosma committed:
Fix calculation issue

jfpva commented on run-brain-structural-pipeline.sh:
```diff
@@ -285,5 +285,7 @@ EOF
 # Set File Permissions
 chmod 0775 -R $OUT_DIR
 
+mv ./slurm-$SLURM_JOBID.out /hpf/projects/cmacgowan/jcosma/4DCardiac/$CASE/SLURM/1-MATLAB-Preproc-i-$SLURM_JOBID.out
```
this should be removed

jfpva commented on run-brain-structural-pipeline.sh:
```diff
@@ -230,5 +285,7 @@ EOF
 # Set File Permissions
 chmod 0775 -R $OUT_DIR
 
+mv ./slurm-$SLURM_JOBID.out /hpf/projects/cmacgowan/jcosma/4DCardiac/$CASE/SLURM/1-MATLAB-Preproc-i-$SLURM_JOBID.out
```
This should be removed.

jfpva committed:
Remove SLURM standard output log file managment

jfpva committed:
Update volume calculations and logging. 

Calculate separate external and total CSF volumes.
Print all label and calculated regions to file.
